tests/test_bookmarks.py

```diff
@@ -485,11 +499,18 @@ def bookmarks_test(self, testable_streams):
                                              msg="A duplicate record was found in the sync for {}\nRECORDS: {}.".format(stream, sync_records))
 
                         sync_record = sync_records[0]
+
                         # Test Workaround Start ##############################
-                        # BUG | https://stitchdata.atlassian.net/browse/SRCE-4975
                         if stream == 'payments':
+
+                            off_keys = MISSING_FROM_SCHEMA[stream] # BUG_2
+                            self.assertParentKeysEqualWithOffKeys(
+                                updated_record, sync_record, off_keys
+                            )
+                            off_keys = PARENT_FIELD_MISSING_SUBFIELDS[stream] | MISSING_FROM_SCHEMA[stream] # BUG_1 | # BUG_2
```

Comment:
I prefer explicitly calling union, not everyone knows `|` means union for sets. But it's not a huge deal.

Reply:
Oh good point. It would be more clear that way. If this PR wasn't blocking Leslie and Zach I would fix it now. I will make a note in the card I spun off to improve readability of these tests.